http/errors.go

```diff
@@ -115,12 +115,14 @@ func (h ErrorHandler) HandleHTTPError(ctx context.Context, err error, w http.Res
 	w.Header().Set("Content-Type", "application/json; charset=utf-8")
 	w.WriteHeader(httpCode)
 	var e struct {
-		Code    string `json:"code"`
-		Message string `json:"message"`
+		Code              string                 `json:"code"`
+		Message           string                 `json:"message"`
+		ValidationErrBody map[string]interface{} `json:"validationErrs,omitempty"`
```

Comment:
change this to Err error `json:"err,omitempty"`, check if you need update the swagger file

Reply:
oh yah, for sure. Wanted to get this API sorted out before I edit the swagger doc